index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300&family=Roboto:ital,wght@0,100;0,300;0,700;1,300;1,500&display=swap" rel="stylesheet">
+  <!-- Material Icons -->
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+  
+<div class = main_section>
+ 
+<header class="nav-bar">    
+  <a href=""><div class="logo-left">My Logo</div></a>
+  <nav><i class="material-icons">menu</i></nav>
+</header>
+
+<section class="headline">
+  <div class="container">
+   <h1>I'm Cookey<br>Glad to see <br> you!</h1>
+    <p>I’m a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+    </p>
```

Comment:
- [ ] [OPTIONAL] Please add a color for this element `color: #344563` to match the Figma design